### DISCUSSION TITLE
Bugs in the appimage

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1140] [Help and Support]
Hey!
There are many bugs in the RE appimage version. For example, I downloaded RE 1.9 as appimage and when I want to create an entitie with /newent [my entitie], it just chat it and don't make anything. When I want to edit my entitie, with number & scroll, it don't anything at all. Can anyone help me to resolve the problems?

**Comment by shacknetisp** [comment]
What is your OS version? (e.g. Ubuntu 18.04)
Please put the contents of your log.txt here so we have more information, you can find it in your [user content directory](https://www.redeclipse.net/docs/FAQ#where-do-i-find-screenshots-logs-and-other-user-data).

**Comment by ghost** [comment]
Iuse fedora 25 workstation edition (linux) and the 1.6.0 version and 1.9.9  as appimage.Am I obliged to put the contents of my log.txt?

**Comment by shacknetisp** [comment]
It is used to show your hardware and catch any other errors that may be occurring. Without it we may not be able to discover the problem.

As for the issue itself, one thing you might try is renaming your config.cfg so that the game starts with a fresh configuration. Also, does your autoexec.cfg have anything in it?

**Comment by ghost** [comment]
I added rainbowchat, authkey, newguis and defaultmap, defalutmode in my autoexec.cfg

**Comment by shacknetisp** [comment]
Ok, perhaps the default input is not going through command processing. Try executing commands by pressing only '/' instead of 'ENTER' and then '/' to open the console input. You could also try the tilde key.
